src/Views/Components/AdminUI/ModalDialog/index.tsx

```diff
@@ -13,21 +13,22 @@ export interface ModalProps {
     handleClose?: MouseEventHandler;
     showHeader?: boolean;
     showCloseIcon?: boolean;
+    wrapperClassName?: string;
 }
 
-export default function Modal
-({
-     title,
-     icon,
-     children,
-     insertInto,
-     handleClose,
-     isOpen = true,
-     showHeader = true,
-     showCloseIcon = true
- }: ModalProps) {
+export default function Modal({
+    title,
+    icon,
+    children,
+    insertInto,
+    handleClose,
+    isOpen = true,
+    showHeader = true,
+    showCloseIcon = true,
+    wrapperClassName = '',
```

Comment:
Do we need to modify the shard component to add a wrapper class, or could we just target styles on the contents of the modal (add a wrapping div in the content)?

I would prefer to not make changes to other domains / shared components unless absolutely necessary (or at least after some attempt to keep the changes inside of the domain).

Reply:
The modal is added with `CreatePortal` directly to the `body` of the page, so it's not a child of any of my components. To avoid styling conflicts with other `DialogModal` components on the page, I thought a better approach optionally setting a custom class to that component.

Reply:
Can we make this work by wrapping the content of the modal with a div and a custom class?

Reply:
We may ultimately need to update the shared component, but I would like to consider another way first.

Reply:
Unfortunately, we can't achieve the same result as the changes I'm applying here are not for its content but changing the modal `max-width` and styling the header.